try {
  let a = 2;
  let b = 4;
  addTwoNumbers(a, b);
} catch (e) {
  console.log(e);
}
